### DISCUSSION TITLE
Can I use both G-Helper & Armoury Crate at the same time?

### DISCUSSION THREAD

**Discussion by ethanic17** [seerge/g-helper#2700] [Q&A] [answered]
I recently just got a 2024 G14 and found out about G-Helper and am loving it. However, I do also still have Armoury Crate installed and was wondering if that would cause any problems or issues from both of them running at the same time. Preferably I would want to keep Armoury Crate just in case but would I be advised to uninstall it since I now have G-Helper? And if I were to uninstall it, is there a specific process of removing all the ASUS services? Thanks.

**Comment by seerge** [comment, marked as answer]
@ethanic17 hello, you can. But in a long term it's better to use something one as otherwise you simply won't know which of the app has set mode or any other settings last. 

Please check readme for details https://github.com/seerge/g-helper/wiki/Requirements

**Reply by ethanic17** [reply]
Ahh okay, I'll probably end up uninstalling Armoury Crate. This wouldn't cause any direct issues or problems correct? Essentially it would just be G-Helper controlling everything right?

Also in the readme it states that I should disable ASUS Smart Display Control; I have it disabled on startup but would it be safe to just uninstall it? Same for MyASUS. Thanks again!

**Reply by seerge** [reply]
@ethanic17 as far as you use official AC uninstaller, it's absolutely fine. You can always install it later if you need it by some reason.

Disabling Smart Display Control from startup is enough. As far as it's not running and not trying to change refresh rate by itself - it's fine :)

MyASUS is up to you.

**Reply by ethanic17** [reply]
Sounds good, thank you! Appreciate your help and all the work you put into this project!

**Comment by NemFeng** [comment]
can i use both g helper and ac for only anime matrix ?, and is there any best way to optimize it?
